ocp-templates/mpp/backend.yaml

```diff
@@ -7,23 +7,24 @@ metadata:
 # Backend
 # ===============================================
 objects:
-- kind: DeploymentConfig
-  apiVersion: v1
+- kind: Deployment
+  apiVersion: apps/v1
   metadata:
-    labels:
-      app: ${APP_NAME}
     name: ibutsu-backend
+    namespace: ${NAMESPACE}
+    annotations:
+      [{"from":{"kind":"ImageStreamTag","name":"ibutsu-backend:latest","namespace":"${NAMESPACE}"},"fieldPath":"spec.template.spec.containers[?(@.name==\"container\")].image","pause":"false"}]
```

Comment:
do we need this?

Reply:
Short answer: yes

The annotations are required for wiring up the Deployment (K8s object) with ImageStreams (Openshift objects). There's [official Openshift docs around this](https://docs.openshift.com/container-platform/4.14/openshift_images/triggering-updates-on-imagestream-changes.html#images-triggering-updates-imagestream-changes-kubernetes-about_triggering-updates-on-imagestream-changes)

Feel free to reach out for a extended version, sorry its not straightforward to explain :)

Reply:
I actually had to add it to the rest of the Deployment objects.